guides/source/active_support_instrumentation.md

```diff
@@ -259,6 +259,32 @@ INFO. The adapters will add their own data as well.
 }
 ```
 
+### connection.active_record
```

Comment:
While I think this use case is valid, I don't think we need to add this to the public API.